src/pages/user/authority/AuthorityList.tsx

```diff
@@ -1,43 +1,98 @@
-import React, { FC } from 'react';
+import React from 'react';
 import { useTranslation } from 'react-i18next';
 import styled from 'styled-components';
+import { Radio, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, Typography } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
 
 import { Authority } from '../../../types/authority.types';
-import AuthorityCard from './AuthorityCard';
 import Label from '../../../components/Label';
+import useFetchLastRegistrationFromAlma from '../../../utils/hooks/useFetchLastRegistration';
+import AffiliationHierarchy from '../../../components/institution/AffiliationHierarchy';
 
-const StyledAuthorityContainer = styled.div`
-  > * {
-    margin-top: 1rem;
-  }
-`;
-
-const StyledClickableDiv = styled.div`
+const StyledTableRow = styled(TableRow)`
   cursor: pointer;
 `;
 
 interface AuthorityListProps {
   authorities: Authority[];
-  searchTerm: string;
-  onSelectAuthority: (authority: Authority) => void;
+  searchTerm?: string;
+  onSelectAuthority?: (authority: Authority) => void;
   selectedArpId?: string;
 }
 
-const AuthorityList: FC<AuthorityListProps> = ({ authorities, searchTerm, onSelectAuthority, selectedArpId }) => {
+const AuthorityList = ({ authorities, searchTerm, onSelectAuthority, selectedArpId }: AuthorityListProps) => {
   const { t } = useTranslation('common');
 
   return (
-    <StyledAuthorityContainer>
-      <Label>{t('search_summary', { count: authorities?.length ?? 0, searchTerm })}</Label>
-      {authorities.map((authority) => (
-        <StyledClickableDiv
-          data-testid="author-radio-button"
-          key={authority.id}
-          onClick={() => onSelectAuthority(authority)}>
-          <AuthorityCard authority={authority} isSelected={selectedArpId === authority.id} />
-        </StyledClickableDiv>
-      ))}
-    </StyledAuthorityContainer>
+    <>
+      {searchTerm && <Label>{t('search_summary', { count: authorities?.length ?? 0, searchTerm })}</Label>}
+
+      <TableContainer>
+        <Table size="small">
+          <TableHead>
+            <TableRow>
+              <TableCell padding="checkbox"></TableCell>
+              <TableCell>{t('name')}</TableCell>
+              <TableCell>{t('last_registration')}</TableCell>
+              <TableCell>{t('profile:heading.organizations')}</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {authorities.map((authority) => (
+              <StyledTableRow
+                data-testid="author-radio-button"
+                key={authority.id}
+                hover
+                onClick={() => onSelectAuthority?.(authority)}
+                selected={authority.id === selectedArpId}>
+                <TableCell padding="checkbox">
```

Comment:
Bare kom til å tenke på en ting; hvorfor har vi en radiobutton her? Kan vi ikke bare klikke på raden? Også blir den forfatteren koblet til? Er det nødvendig å først trykke på radioknappen og deretter «koble til»? 🤔

Reply:
Jeg tenker det er litt mer fail safe slik da. Hvis man trykker på feil rad må man fortsatt aktivt bekrefte før det skjer noe.
Radioknappen er vel egentlig bare for å vise enda klarere at man skal trykke på en rad, men mulig det ikke trengs ja. 
det er ingen funksjonalitet i radio button nå, så vi kan nevne det for Kine, så er det en smal sak å evt fjerne den kolonna senere